### DISCUSSION TITLE
Up next dialog

### DISCUSSION THREAD

**Discussion by heyhippari** [jellyfin/jellyfin-vue#158] [Features]
Depends on #74

**Comment by AlfHou** [comment]
Is this the component being refered to?
![Screenshot 2020-11-05 at 19 31 14](https://user-images.githubusercontent.com/20001253/98281865-a300d780-1f9d-11eb-94ef-fe876872b0e7.png)

**Comment by AlfHou** [comment]
Or this one ?
![image](https://user-images.githubusercontent.com/20001253/98282034-e52a1900-1f9d-11eb-8126-3ca66dc490b9.png)

**Comment by camc314** [comment]
The latter. It appears towards the end of an episode

**Comment by AlfHou** [comment]
Great! I'm guessing we want something very similar? Not like a full page dialog?

**Comment by camc314** [comment]
Yeah something fairly similar

It was also reworked in:
https://github.com/jellyfin/jellyfin-web/pull/1825

To look a bit more modern

**Comment by AlfHou** [comment]
I am trying to work on this.
I have quite a lot of school coming up, so can't promise anything but I've started on it

**Comment by dkanada** [comment]
I would also suggest a toggle to make it extremely small, like the one Netflix uses. Even the new one takes up more space than I'd prefer.

**Comment by AlfHou** [comment]
@dkanada I'm correct in assuming you would toggle the size in the settings right?

**Comment by camc314** [comment]
Yeah that's correct
Currently, there is settings area, so creating both a smaller and larger dialog, with the ability to switch in the js would be fine for now.

**Comment by heyhippari** [comment]
Do note that we probably need the playback manager for the implementation of this to make sense, since we don't have any queue support at the moment, and this needs a queue in order to know what the next item will be.
